File: support_tools/swigwin-3.0.12/Examples/test-suite/go/li_cmalloc_runme.go
package main

import . "./li_cmalloc"

func main() {
	p := Malloc_int()
	Free_int(p)

	ok := false
	func() {
		defer func() {
			if recover() != nil {
				ok = true
			}
		}()
		p = Calloc_int(-1)
		if p == nil {
			ok = true
		}
		Free_int(p)
	}()
	if !ok {
		panic(0)
	}
}

File: support_tools/swigwin-3.0.12/Examples/test-suite/octave/li_cmalloc_runme.m
# do not dump Octave core
if exist("crash_dumps_octave_core", "builtin")
  crash_dumps_octave_core(0);
endif

li_cmalloc

p = malloc_int();
free_int(p);

ok = 0;
try
    p = calloc_int(-1);
    free_int(p);
catch
    ok = 1;
end_try_catch

if (ok != 1)
    error
endif

